docs/tutorials/python/tutorial_scripts/evaluation.py

```diff
@@ -0,0 +1,71 @@
+"""
+Here is where you'll find the code for the Evaluation tutorial.
+"""
+
+from synapseclient import Synapse
+from synapseclient.models import Evaluation, Project
+
+syn = Synapse()
+syn.login()
+
+# REQUIRED: Set this to the Synapse user ID or team ID you want to grant permissions to
+# Do NOT leave this as None - the script will not work properly
+PRINCIPAL_ID = None  # Replace with actual user/team ID
+
+# Retrieve the Project where your Evaluation will be stored
+project = Project(name="My uniquely named project about Alzheimer's Disease").get()
+project_id = project.id
+
+print(f"Working within Project: {project_id}")
+
+# Create a new Evaluation object
+evaluation = Evaluation(
+    name="My Challenge Evaluation - Round 1",
+    description="Evaluation for my data challenge",
+    content_source=project_id,
```

Comment:
Although the Rest API Calls this `contentSource`, is this is most clear attribute name to use? Since we map fields from/to the Python classes we can have a slight divergence in naming if it makes the experience better.